tests/search/test_wrapper.py

```diff
@@ -0,0 +1,40 @@
+from mock import Mock
```

Comment:
Ignoring until a new pull request for this work.

I am a bit confused on how I will use this means to comment since it won't let me comment on lines that were not changed